ui/packages/catalog-ui-search/src/main/webapp/component/alert/alert.js

```diff
@@ -21,6 +21,16 @@ const router = require('../router/router.js')
 const user = require('../singletons/user-instance.js')
 const cql = require('../../js/cql.js')
 
+const sourceDataFromAlert = alert => {
+  const sources = alert.get('src')
+  const haveSources = sources && sources.length > 0
+
+  return {
+    federation: (haveSources && 'selected') || 'enterprise',
```

Comment:
```suggestion
    federation: haveSources ? 'selected' : 'enterprise'
```
Probably just not what I'm used to, but it seems strange to write the logic this way versus using a ternary